Directory-Structure.md

```diff
@@ -1,6 +1,13 @@
 Not all files/directories might be available, depending on your setup
 
-## Linux
+The easiest way to see the root folder is to click `Folders` > `View Instance Folder`. Then, go up one level.
```

Comment:
Many people don't understand what is meant with one level. Maybe say parent folder or something. Ideally would be images ofc